test/lib/usd/translators/testUsdExportImportRoundtripPreviewSurface.py

```diff
@@ -116,37 +120,37 @@ def testUsdPreviewSurfaceRoundtrip(self):
         # Check the new sphere is in the new shading group:
         self.assertTrue(cmds.sets(
             "pSphere1Shape",
-            isMember="USD_Materials:usdPreviewSurface1SG"))
+            isMember=material_sg))
 
         # Check that we have no spurious "Looks" transform
-        expectedTr = set(['front', 'persp', 'side', 'top', 'pSphere1', 'pSphere2'])
+        expectedTr = set(['front', 'persp', 'side', 'top', 'pSphere1'])
```

Comment:
What happened with pSphere2?

Reply:
I added a file->new to make sure no renaming would happen on import. This means that after the roundtrip there is only one sphere in the final scene and it needs to have the same name it had at export time.